package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@fastify/swagger-ui",
-  "version": "1.6.0",
+  "version": "1.7.0",
```

Comment:
Please revert this, I will bump it on release

Reply:
This is from @Eomm but I can revert it.

Reply:
Please do, thx

Reply:
You might need to rebase your changes